scripts/resume.js

```diff
@@ -0,0 +1,44 @@
+const resume = {
+  basics: {
+    profiles: []
+  },
+
+  // Add Basic Contact Information
+  addBasics: function(basicsObj) {
+    // save the previous profiles, so it does't overwrite
```

Comment:
I personally prefer the ES6+ syntax for defining methods, like this: `addBasics(basicsObj) {}`